### PR TITLE
Capitalize SSL and remove slashes from rpm variables

### PR DESCRIPTION
**Pull request by felixsch**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ikapelyukhin** [comment]
`%define conf_dir %{_sysconfdir}/rmt` <--- just remove the slashes here.

**Comment by ikapelyukhin** [comment]
I would suggest double-checking the whole spec file though, in case there are some missing slashes somewhere else.

**Comment by Sergeykot** [comment]
FYI: the build is failing. error looks like  `File not found:  ..../etc/rmt`

File: package/rmt-server.spec [hunk 1]
```diff
@@ -102,7 +102,7 @@ bundle.ruby2.5 install %{?jobs:--jobs %{jobs}} --without test development --depl
 mkdir -p %{buildroot}%{data_dir}
 mkdir -p %{buildroot}%{lib_dir}
 mkdir -p %{buildroot}%{app_dir}
-mkdir -p %{buildroot}%{conf_dir}ssl
+mkdir -p %{buildroot}%{conf_dir}/ssl
 mkdir -p %{buildroot}var/lib/rmt
```
Comment (ikapelyukhin) [inline package/rmt-server.spec]: This creates `/home/abuild/rpmbuild/BUILDROOT/rmt-server-1.0.6-0.x86_64var/lib/rmt`.

Needs to be removed, there's already `mkdir -p %{buildroot}%{lib_dir}`, which creates `/home/abuild/rpmbuild/BUILDROOT/rmt-server-1.0.6-0.x86_64/var/lib/rmt`.

**Pull request merged**